tuplex/io/include/orc/DictBatch.h

```diff
@@ -0,0 +1,180 @@
+//--------------------------------------------------------------------------------------------------------------------//
+//                                                                                                                    //
+//                                      Tuplex: Blazing Fast Python Data Science                                      //
+//                                                                                                                    //
+//                                                                                                                    //
+//  (c) 2017 - 2021, Tuplex team                                                                                      //
+//  Created by Ben Givertz first on 8/31/2021                                                                         //
+//  License: Apache 2.0                                                                                               //
+//--------------------------------------------------------------------------------------------------------------------//
+
+#ifndef TUPLEX_DICTBATCH_H
+#define TUPLEX_DICTBATCH_H
+
+namespace tuplex { namespace orc {
+
+/*!
+ * Implementation of OrcBatch for tuplex Dictionary types.
+ */
+class DictBatch : public OrcBatch {
+public:
+    DictBatch(::orc::ColumnVectorBatch *orcBatch,
```

Comment:
delete Default constructor or add and initialize elements. This applies to all batches.